CodeStyleExercise/ColorsViewController.swift

```diff
@@ -8,55 +8,52 @@
 
 import UIKit
 
-class ColorsViewController : UIViewController, UICollectionViewDataSource, UITextFieldDelegate
+class ColorsViewController :
+    UIViewController,
+    UICollectionViewDataSource,
+    UITextFieldDelegate
 {
-
-    class constants {
-        static let DFLTNUMCOLORS:Int=12
-    }
-
+    
     @IBOutlet weak var collectionView:UICollectionView!
     @IBOutlet weak var colorCountTextField: UITextField!
 
-    var ct: Int? = 12
+    var colorCount = 12
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.collectionView.register(UINib(nibName: "ColorsCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "Cell")
-
+        self.collectionView.register(UINib(nibName: "ColorsCollectionViewCell",
```

Comment:
Strings should not be hardcoded in but either stored as a constant in the class they're used in or, in the case of cell reuse identifiers, they can be stored as a static constant called `defaultIdentifier` and accessed that way.